cli_tools/gce_vm_image_import/importer/inflater.go

```diff
@@ -60,19 +177,50 @@ func (inflater daisyInflater) inflate(ctx context.Context) (persistentDisk, erro
 	targetSizeGB := inflater.wf.GetSerialConsoleOutputValue("target-size-gb")
 	sourceSizeGB := inflater.wf.GetSerialConsoleOutputValue("source-size-gb")
 	importFileFormat := inflater.wf.GetSerialConsoleOutputValue("import-file-format")
+	checksum := inflater.wf.GetSerialConsoleOutputValue("disk-checksum")
 	return persistentDisk{
-		uri:        inflater.inflatedDiskURI,
-		sizeGb:     string_utils.SafeStringToInt(targetSizeGB),
-		sourceGb:   string_utils.SafeStringToInt(sourceSizeGB),
-		sourceType: importFileFormat,
+		uri:           inflater.inflatedDiskURI,
+		sizeGb:        stringUtils.SafeStringToInt(targetSizeGB),
+		sourceGb:      stringUtils.SafeStringToInt(sourceSizeGB),
+		sourceType:    importFileFormat,
+		checksum:      checksum,
+		inflationTime: time.Since(startTime),
 	}, nil
 }
 
 type persistentDisk struct {
-	uri        string
-	sizeGb     int64
-	sourceGb   int64
-	sourceType string
+	uri                 string
+	sizeGb              int64
+	sourceGb            int64
+	sourceType          string
+	checksum            string
+	inflationTime       time.Duration
+	shadowInflationTime time.Duration
+	matchResult         string
```

Comment:
These are temporary for the experiment, right? If so, can you add a comment mentioning that?

Reply:
done